controllers/nnf_workflow_controller_helpers.go

```diff
@@ -1164,7 +1164,7 @@ exit 1
 			Name:  fmt.Sprintf("mpi-wait-for-worker-%d", worker),
 			Image: spec.Containers[0].Image,
 			Command: []string{
-				"/bin/bash",
+				"/bin/sh",
```

Comment:
This actually breaks the script because using `sh` here rather than `bash` causes the script to bomb out when `nslookup` fails instead of looping.

I guess I'm still confused how using bash caused a failure for you here.

Reply:
The container alpine:latest, when pulled for my M1 mac, does not have a 'bash' executable anywhere.

```
: dean-mbp; docker run -it --rm --name c1 alpine:latest sh               
Unable to find image 'alpine:latest' locally
latest: Pulling from library/alpine
08409d417260: Pull complete 
Digest: sha256:02bb6f428431fbc2809c5d1b41eab5a68350194fb508869a33cb1af4444c9b11
Status: Downloaded newer image for alpine:latest
/ # ls
bin    etc    lib    mnt    proc   run    srv    tmp    var
dev    home   media  opt    root   sbin   sys    usr
/ # type bash
bash: not found
/ # ls /bin/bash
ls: /bin/bash: No such file or directory
/ # type sh
sh is /bin/sh
/ # ls /usr/bin/bash
ls: /usr/bin/bash: No such file or directory
/ # exit
```

Reply:
So that for-loop doesn't work in 'sh':
```
: dean-mbp; docker run -it --rm --name c1 alpine:latest sh
/ # for i in {1..5}; do echo $i; done
{1..5}
```


I'll put up another PR to switch it to use `seq`:
```
/ # for i in $(seq 1 5); do echo $i; done
1
2
3
4
5
```

Reply:
I see this same lack of bash when I pull it to dp1a:
```
[root@rabbit-k8s-master ~]# docker run -it --rm --name c1 alpine:latest sh
Unable to find image 'alpine:latest' locally
latest: Pulling from library/alpine
8a49fdb3b6a5: Pull complete 
Digest: sha256:02bb6f428431fbc2809c5d1b41eab5a68350194fb508869a33cb1af4444c9b11
Status: Downloaded newer image for alpine:latest
/ # type bash
bash: not found
/ # ls /bin/bash
ls: /bin/bash: No such file or directory
/ # 
```

Reply:
So it's not an issue with the M1 builds.

Reply:
PR posted in https://github.com/NearNodeFlash/nnf-sos/pull/201